installer/templates/phx_assets/webpack/app.js

```diff
@@ -15,3 +15,8 @@ import css from "../css/app.css"
 //
 // Local files can be imported directly using relative paths, for example:
 // import socket from "./socket"
+
+<%= if live do %>// Enabled LiveView Socket
+import LiveSocket from "phoenix_live_view"
+let liveSocket = new LiveSocket("/live")
+liveSocket.connect()<% end %>
```

Comment:
`--live` was explicitly asked for, so I left this code directly commented out. When moving towards `--live` as the default, we possibly might want this shown, but commented out.